migrations/database_migration.py

```diff
@@ -0,0 +1,136 @@
+            overall_profits = json.loads(user_overall_profits)
+            week_profits = json.loads(user_week_profits)
+
+            last_week_user_balances = {}
+            last_week_user_overall_profits = {}
+
+            for tg_id, overall_profit in overall_profits.items():
+                last_week_user_balances[tg_id] = overall_profit
+
+            for tg_id, week_profit in week_profits.items():
+                last_week_user_overall_profits[tg_id] = week_profit
+
+            user_overall_profits_percent, user_current_week_profits_percent = calculate_week_user_profits_percent(
+                overall_balance, last_week_user_balances, last_week_user_overall_profits
+            )
+
+            for tg_id, overall_profit_percent in user_overall_profits_percent.items():
+                tg_id = int(tg_id)
+                overall_profit_percent = float(overall_profit_percent)
+
+                existing_week = pg_session.query(UserWeekStat).filter_by(date=date, tg_id=tg_id).first()
+                if existing_week:
+                    existing_week.balance = overall_balance
+                    existing_week.overall_profit = overall_profit
+                    existing_week.overall_profit_percent = overall_profit_percent
+                else:
+                    pg_session.add(UserWeekStat(date=date, tg_id=tg_id, balance=overall_balance,
+                                                overall_profit=overall_profit,
+                                                overall_profit_percent=overall_profit_percent,
+                                                week_profit=0,  # default value
+                                                week_profit_percent=0  # default value
+                                                ))
+
+            for tg_id, week_profit_percent in user_current_week_profits_percent.items():
+                tg_id = int(tg_id)
+                week_profit_percent = float(week_profit_percent)
+
+                existing_week_profit = pg_session.query(UserWeekStat).filter_by(date=date, tg_id=tg_id).first()
+                if existing_week_profit:
+                    existing_week_profit.week_profit = week_profit
+                    existing_week_profit.week_profit_percent = week_profit_percent
+                else:
+                    pg_session.add(UserWeekStat(date=date, tg_id=tg_id, week_profit=week_profit,
+                                                week_profit_percent=week_profit_percent))
+
+            pg_session.commit()
+            print(f"Transferred data for telegram_tag: {telegram_tag}")
+    except Exception as e:
+        # Rollback transaction if error occurs
+        pg_session.rollback()
+        print(f"An error occurred: {e}")
+
+    finally:
+        # Close connections
+        sqlite_session.close()
+        pg_session.close()
+
+
+if __name__ == "__main__":
```

Comment:
All your scripts must be able to run from CLI (such as `poetry run python3 script.py`). You can not make sure your scripts will be run using PyCharm or other IDEs.